packages/ui-navigation/package.json

```diff
@@ -46,6 +46,8 @@
     "@instructure/ui-tooltip": "^7.3.1",
     "@instructure/ui-utils": "^7.3.1",
     "@instructure/ui-view": "^7.3.1",
+    "@instructure/emotion": "^7.3.1",
+    "@instructure/ui-themes": "^7.3.1",
```

Comment:
@matyasf pointed out that this dependency is only needed for tests, so we can move it to "devDependences" (here and all future components, where it is needed for theme.test.js)